docs/developer-docs/integrations/internet-identity/integrate-identity.md

```diff
@@ -244,6 +229,12 @@ Once you save it, select the 'I saved it, continue' button.
 
 ![Local integration 4](../_attachments/II_greet_2.png)
 
+### Local Frontend Development
+
+When modifying this examples frontend try to develop against a deployment on a local development server instead of using the canister deployed frontend. This will enable Hot Module Reloading, allowing you to see any modifications made to your frontend instantaneously.
```

Comment:
```suggestion
When modifying this example's frontend, it is recommended to develop using a local development server instead of using the deployed frontend canister. This is because using a local develop server will enable Hot Module Reloading, allowing you to see any modifications made to your frontend instantaneously, rather than having to redeploy the frontend canister to see the changes. 
```